README.md

```diff
@@ -100,9 +100,9 @@ public class Startup
                 .UseMySql(connectionString, serverVersion)
                 // The following three options help with debugging, but should
                 // be changed or removed for production.
-                .LogTo(Console.WriteLine, LogLevel.Information);
+                .LogTo(Console.WriteLine, LogLevel.Information)
                 .EnableSensitiveDataLogging()
-                .EnableDetailedErrors()
+                .EnableDetailedErrors());
```

Comment:
The correct way to fix this is to just remove the semicolon from the end of the `LogTo()` line.
Nothing else has to be added.

Reply:
@lauxjpn  Thanks for the heads up, fixed!